controls/mousestates/cMouseState.h

```diff
@@ -0,0 +1,32 @@
+#ifndef D2TM_CMOUSESTATE_H
+#define D2TM_CMOUSESTATE_H
+
+#include "observers/cInputObserver.h"
+#include "controls/cMouse.h"
+//#include "controls/cGameControlsContext.h"
+
+// forward declaration
+class cGameControlsContext;
+class cPlayer;
```

Comment:
Yes, cPlayer is hard to include. I guess for the time being, we'll have to forward declare the class everywhere.